### PR TITLE
feat: add batchGetEffectiveIamPolicies sample code.

### LINKED ISSUE #653: feat: add batchGetEffectiveIamPolicies sample code.
Thanks for stopping by to let us know something could be better!

Add batchGetEffectiveIamPolicies sample code and also lint the protobuf
imports.

### PR DESCRIPTION
**Pull request by aaronlichen-hp**
Add batchGetEffectiveIamPolicies sample code and also lint the protobuf
imports.

Fixes #653 🦕

### CODE REVIEW

File: samples/getBatchEffectiveIamPolicies.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    const projectId = await client.getProjectId();
+    const request = {
+      scope: `projects/${projectId}`,
+      names: assetNames.split(','),
+    };
+
+    // Handle the operation using the promise pattern.
+    const result = await client.batchGetEffectiveIamPolicies(request);
+    // Handle the the response.
+    console.log(util.inspect(result, {depth: null}));
```
Comment (danielbankhead) [inline samples/getBatchEffectiveIamPolicies.js:39]: No need for the `util.inspect`; `console.dir` would be an appropriate alternative here
```suggestion
    console.dir(result);
```

File: samples/test/sample.test.js [hunk 2]
```diff
@@ -186,6 +186,18 @@ describe('quickstart sample tests', () => {
...
 
+  it('should get effective iam policies successfully', async () => {
+    const assetName = `//storage.googleapis.com/${bucketName}`;
+    let waitMs = 1000;
+    let included = false;
+    for (let retry = 0; retry < 3 && !included; ++retry) {
+      await sleep((waitMs *= 2));
+      const stdout = execSync(`node getBatchEffectiveIamPolicies ${assetName}`);
+      included = stdout.includes(assetName);
+    }
```
Comment (danielbankhead) [inline samples/test/sample.test.js:197]: Why would this need to retry? The `execSync` should fail or contain the desired output in one call

**Review by danielbankhead — CHANGES_REQUESTED** [review]

File: samples/getBatchEffectiveIamPolicies.js (hunk 1 above)
Comment (aaronlichen-hp) [reply samples/getBatchEffectiveIamPolicies.js:39]: Done

File: samples/test/sample.test.js (hunk 2 above)
Comment (aaronlichen-hp) [reply samples/test/sample.test.js]: Done

**Review by danielbankhead — APPROVED** [review]

File: samples/getBatchEffectiveIamPolicies.js (hunk 1 above)
Comment (aaronlichen-hp) [reply samples/getBatchEffectiveIamPolicies.js]: I think I will need this to unwrap the nested result fields. Reverting back.

File: samples/getBatchEffectiveIamPolicies.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    const projectId = await client.getProjectId();
+    const request = {
+      scope: `projects/${projectId}`,
+      names: assetNames.split(','),
+    };
+
+    // Handle the operation using the promise pattern.
+    const result = await client.batchGetEffectiveIamPolicies(request);
+    // Handle the the response.
+    console.dir(result);
```
Comment (danielbankhead) [inline samples/getBatchEffectiveIamPolicies.js:38]: Apologies, let's add `{depth: null}` here
```suggestion
    console.dir(result, {depth: null});
```

**Pull request merged**